drivers/gles3/rasterizer_storage_gles3.cpp

```diff
@@ -6905,7 +6905,11 @@ void RasterizerStorageGLES3::initialize() {
 	config.s3tc_supported = config.extensions.has("GL_EXT_texture_compression_dxt1") || config.extensions.has("GL_EXT_texture_compression_s3tc") || config.extensions.has("WEBGL_compressed_texture_s3tc");
 	config.etc_supported = config.extensions.has("GL_OES_compressed_ETC1_RGB8_texture");
 	config.latc_supported = config.extensions.has("GL_EXT_texture_compression_latc");
-	config.rgtc_supported = config.extensions.has("GL_EXT_texture_compression_rgtc");
+#ifdef OSX_ENABLED
+	config.rgtc_supported = true;
```

Comment:
Currently macOS max. OpenGL version is 4.1 and all other used extensions aren't core in 4.1.
BPTC is core in 4.2 and may have similar issue in case macOS OpenGL driver will be updated.